extensions/ecg-dicom/.webpack/webpack.prod.js

```diff
@@ -1,9 +1,10 @@
 const path = require('path');
 const pkg = require('../package.json');
 
-const outputFile = 'index.umd.js';
 const rootDir = path.resolve(__dirname, '../');
-const outputFolder = path.join(__dirname, '../public/umd', pkg.name);
+const outputFolder = path.join(__dirname, '../dist');
+const packageName = pkg.name.replace('/', '--');
+const outputFile = `index.umd.js`;
```

Comment:
Seems like the outputFile also needs to be ${packageName}.umd.js, otherwise modes and extensions will both write to index.umd.js